C/Chuffed/build_tarballs.jl

```diff
@@ -2,31 +2,28 @@ using BinaryBuilder, Pkg
 
 name = "Chuffed"
 
-version = v"0.10.4"
+version = v"0.13.2"
 
 sources = [
     GitSource(
         "https://github.com/chuffed/chuffed.git",
-        "23b9fcee3bb30b11f68d82ef4534040ebae1a8fb",
+        "2016f7eb7943a86b9ce93bb70b821d701667a5ca",
     ),
 ]
 
 script = raw"""
 cd $WORKSPACE/srcdir/chuffed
-mkdir -p build
-cd build
-cmake \
+cmake -B build \
     -DCMAKE_INSTALL_PREFIX=${prefix} \
     -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
-    -DCMAKE_BUILD_TYPE=Release \
-    ..
+    -DCMAKE_BUILD_TYPE=Release
```

Comment:
```suggestion
    -DCMAKE_BUILD_TYPE=Release \
    -DCP_PROFILER=OFF
```
It looks like it is just coming from the optional profiler. Let's turn that off.